src/Pages/login.jsx

```diff
@@ -0,0 +1,111 @@
+import { event } from "jquery";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default () => {
+  const navigate = useNavigate();
+  const [userName, setUserName] = useState(null);
+  const [password, setPassword] = useState(null);
+
+  const [isUsernameActivated, setUsernameActivated] = useState(false);
+  const [isPasswordActivated, setPasswordActivated] = useState(false);
+
+  const [unalert, setUnAlert] = useState(null);
```

Comment:
This is a good example of naming variables. We should always look to use meaningful names. As a component grows it helps a lot to understand logic when there are meaningful names associated. For example this should have been something like `userNameAlert`, which clearly explains the value it holds.

And the other thing is we should always use lower camel case for variables for readability. `unAlert` is better than `unalert`.

Reply:
Fixed e97eae75b859e2f1f70c09d6abe315d63c384f20